lib/brainstem/controller_methods.rb

```diff
@@ -36,7 +36,7 @@ def brainstem_present(name, options = {}, &block)
     #                           only required if the name cannot be inferred.
     # @return (see PresenterCollection#presenting)
     def brainstem_present_object(objects, options = {})
-      options.merge!(:params => params, :apply_default_filters => false)
+      options.merge!(:params => params.to_unsafe_h, :apply_default_filters => false)
```

Comment:
These concern me, but I don't have a better idea.

Reply:
Because they’re only used for filtering, and not assigned to a model, I think it’s ok.